br/br-snapshot-architecture.md

```diff
@@ -0,0 +1,160 @@
+---
+title: TiDB Snapshot Backup and Restore Architecture
+summary: Learn about the architecture of TiDB snapshot backup and restore.
+---
+
+# TiDB Snapshot Backup and Restore Architecture
+
+This document introduces the architecture and process of TiDB snapshot backup and restore using the br command-line tool (hereinafter referred to as `br`) as an example.
```

Comment:
br cli is just a use interface, it can not represent design the of the kernel, just like the tidb operator backup is another interface